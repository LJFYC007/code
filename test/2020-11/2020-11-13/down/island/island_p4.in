50 200 10 50
8 3 4 5 1 5 2 8 6 10 10 1 7 5 6 10 5 10 4 6 1 8 10 2 7 7 3 9 10 9 7 7 1 10 3 3 6 4 3 2 6 2 4 2 8 1 1 2 10 5 
897 450 904 186 173 122 344 898 280 123 6 396 611 419 317 313 181 504 259 953 707 866 926 220 297 963 220 370 774 271 956 23 72 859 560 245 332 256 142 611 730 499 358 341 917 26 5 450 881 615 
2 8 10 10 9 7 2 9 8 8 1 3 2 3 3 1 9 4 6 2 6 6 10 5 8 7 10 2 6 10 7 9 9 8 8 8 6 2 8 3 9 8 7 10 10 9 2 8 4 8 
5 24 2 3 14 20 13 45 28 34 
1 2
2 3
1 4
3 5
4 6
3 7
6 8
1 9
6 10
6 11
5 12
7 13
8 14
5 15
5 16
4 17
8 18
12 19
14 20
1 21
15 22
7 23
16 24
18 25
5 26
24 27
12 28
7 29
4 30
14 31
14 32
3 33
33 34
30 35
33 36
19 37
24 38
35 39
20 40
27 41
28 42
28 43
43 44
30 45
9 46
18 47
21 48
39 49
48 50
11 35
22 39
32 23
18 29
44 10
3 49
35 25
47 22
37 28
13 27
41 32
12 14
21 3
24 44
47 8
35 7
44 8
45 27
32 15
5 27
26 10
25 13
36 24
34 25
1 49
1 43
30 13
8 2
17 32
45 14
20 35
41 17
11 23
33 18
49 10
29 26
22 16
49 6
40 1
6 41
43 35
5 50
36 21
33 32
34 26
15 6
24 22
30 8
41 28
17 21
3 39
37 3
46 26
3 1
37 24
49 24
47 31
5 30
6 20
37 17
27 12
19 33
4 38
26 42
42 21
19 47
23 39
43 9
12 41
34 8
40 31
9 28
48 35
42 8
22 27
14 22
20 7
42 39
3 16
27 46
26 38
38 9
48 11
49 37
44 7
14 43
44 5
32 37
13 5
13 9
34 28
31 3
35 24
41 37
40 19
34 15
9 22
24 6
34 24
44 27
32 9
19 27
14 2
14 26
9 26
36 42
4 16
46 38
40 39
24 31
7 8
47 15
31 22
45 13
43 27
22 13
3 35
17 16
12 25
44 47
34 6
3 9
36 10
18 32
24 50
4 46
16 50
10 8
26 31
17 38
46 30
14 39
29 31
37 43
46 22
31 35
9 50
16 33
50 21
30 15
21 39
24 46
20 45
28 38
33 24
20 48
14 48
31 1
22 41
25 4
25 36
1 140537670 5 4 
1 270186888 8 4 
1 161229107 7 4 
0 802834703 38 26
0 31349708 1 40
0 722074739 14 31
0 930324182 30 35
0 196764383 5 50
0 561557219 9 50
0 280621358 24 49
1 592699457 3 4 
1 828051606 3 4 
0 601747779 24 49
1 611969728 4 4 
0 372798148 22 24
1 884117596 10 4 
1 999291160 10 4 
0 200501336 13 30
0 187537485 24 31
0 668275415 28 38
0 790760678 24 37
0 356163904 27 19
0 782074541 28 41
0 178194127 5 44
0 426008839 47 22
0 13836836 35 39
0 6042810 26 46
1 234846011 7 4 
1 995230087 10 4 
0 330487312 9 22
1 598737825 7 4 
0 343046938 13 9
0 550291503 26 14
0 464182916 14 20
0 743754846 34 8
0 612852384 5 13
0 536178508 30 46
0 63711520 30 15
0 150911138 37 28
0 104770970 48 21
0 676559185 36 10
0 2670300 45 27
0 994304322 14 31
0 646089581 15 32
0 96335181 23 7
0 494179293 33 36
1 101553490 3 4 
0 667076748 31 29
0 98594962 14 31
0 44332474 10 8
