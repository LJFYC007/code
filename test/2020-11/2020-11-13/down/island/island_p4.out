6 5 3 2 0 0 18 6 2 8 
